packages/blockchain-wallet-v4-frontend/src/scenes/Interest/IntroCard/index.tsx

```diff
@@ -43,6 +46,65 @@ const IneligibleBanner = styled.div`
 class IntroCard extends PureComponent<
   ParentStateType & Props & SuccessStateType
 > {
+  renderAdditionalInfo = () => {
+    const { analyticsActions, interestRateArray } = this.props
+    const highestRate = interestRateArray.reduce((a, b) => Math.max(a, b))
+    return (
+      <BoxStyledAdditional>
+        <ContentWrapper>
+          <IconWrapper>
+            <Icon color='blue600' name='percentage' size='32px' />
+          </IconWrapper>
+          <Text
+            size='20px'
+            color='grey800'
+            weight={600}
+            style={{ marginTop: '16px' }}
+          >
+            <FormattedMessage
+              id='scenes.interest.additional_info_required'
+              defaultMessage='Additional Info Required'
+            />
+          </Text>
+          <Text
+            size='14px'
+            color='grey600'
+            weight={500}
+            style={{ marginTop: '4px', lineHeight: 1.5 }}
+          >
+            <FormattedMessage
+              id='scenes.interest.additional_info_required_description'
+              defaultMessage='Please supply the additional information required to avoid delays on withdrawal.'
+              values={{ highestRate }}
```

Comment:
I don't see the `highestRate` variable in the message text. Is it needed?

Reply:
yeah this is just WIP PR, since I have to wait for BE and didn't wanted to keep changes on my local branch ;) will cleanup

Reply:
Is it fixed?

Reply:
yup